packages/web/src/components/PageBanner.tsx

```diff
@@ -26,7 +27,12 @@ const PageBanner = ({ children, color, backButtonAction }: Props) => {
         <Grid item>
           <BackButton backButtonAction={backButtonAction} />
         </Grid>
-        <Grid item>{children}</Grid>
+
+        <List>
+          <ListItem>
+            <Grid item>{children}</Grid>
+          </ListItem>
+        </List>
```

Comment:
Ya, I think using a MUI list here is appropriate 👍 We could follow [this example](https://github.com/jakubroztocil/rrule/issues/13) and also implement the `<BackButton/>` wrapped in a `<ListItemIcon/>`. I think that would mean that we could get rid of the `<Grid/>` wrapping this component too!

Reply:
I tried that, on this {packages/web/src/components/PageBanner.tsx} wrapping the <BackButton/> in the List and <ListItemIcon/>  but the result was not the spacing that the Issue was asking. Do you still want me to update it?

Reply:
Not sure 🤔 Is that due to the extra spacing that is directly on the `<BackButton/>` component?

Reply:
Looks like this has changed, so I'll resolve this and start a comment on the updates!